src/positional.rs

```diff
@@ -225,4 +252,35 @@ mod tests {
     search_directory: None,
     arguments: ["a", "a=b"],
   }
+
+  test! {
```

Comment:
I think these would be better as integration tests. I'd also add one that tries other args, like `.`, `./foo`, `../foo`

Reply:
With unit tests, you have to think about whether or not the unit test captures all aspects of the feature or change. In this case, the error test doesn't check the formatting of the error message, which an integration test would. But, engaging my brain for a second, I think it's fine here.